account/db.go

```diff
@@ -0,0 +1,150 @@
+// Package account is a lightweight account system for Replicache customers.
+package account
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/attic-labs/noms/go/datas"
+	"github.com/attic-labs/noms/go/marshal"
+	"github.com/attic-labs/noms/go/types"
+)
+
+// DB represents the Replicache account database. It is modeled on the pattern
+// established by the main db (db/db.go).
+type DB struct {
+	ds datas.Dataset
+
+	mu   sync.Mutex
+	head Commit
+}
+
+// Commit is the git-like commit structure noms uses to store values.
+// The account database keeps a single Commit at the head of its dataset
+// as opposed to a chain of Commits representing changes. Noms requires
+// a Commit structure like this.
```

Comment:
```suggestion
// Commit is the Git-like commit structure Noms uses to store values.
// The account database keeps a single Commit at the head of its dataset
// containing all current entries.
```

This is the way all Noms databases work, so it's a little confusing to have the "as opposed" bit.